docs/DeveloperGuide.md

```diff
@@ -252,6 +252,80 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Adding a task
+
+Syntax: `addtask t/TITLE [c/CONTENT] [st/STATUS]` </br>
+Purpose: Allows users to add tasks into the OfficeConnectModel
+
+#### Implementation
+The implementation of this feature is supported by `AddTaskCommand` and `AddTaskCommandParser`.
+Below is an activity diagram that illustrates how a user adds new tasks into the OfficeConnectModel. </br>
+
+![AddTaskSequenceDiagram](images/AddTaskSequenceDiagram.png)
+
```

Comment:
The arrow representing the return values in your sequence diagrams should be indicated with a dotted line instead of a solid line. You might also want to remove the instance entities at the bottom of the diagram. There should also be a ":" in front of the component labels as they represent instances of those classes.

Reply:
great catch, thanks for the suggestions!